tests/test_conjuring.py

```diff
@@ -4,6 +4,7 @@
 from invoke import Collection
 
 from conjuring.grimoire import collection_from_python_files, magically_add_tasks
+from conjuring.visibility import ShouldDisplayTasks, is_poetry_project
```

Comment:
Is this line being used?

The tests are currently... hacky. 😅

Reply:
Forgot to commit test, fixed. Now only for positive case though

Reply:
> Now only for positive case though

Good enough. 😉 

The test suite is not ideal right now.
But it's always good to have a few tests.

This project has a lot of room for improvement and [nitpick](https://github.com/andreoliwa/nitpick)ing.